src/main/java/org/ow2/proactive/workflow_catalog/rest/entity/WorkflowRevision.java

```diff
@@ -241,4 +241,8 @@ public String getLayout() {
         return layout;
     }
 
+    @Override
+    public int compareTo(Object o) {
+        return createdAt.compareTo(((WorkflowRevision) o).createdAt);
```

Comment:
The TreeSet that is used to sort the revisions (once Hibernate fetches them) expects the embedded items to be Comparable, and since I only need them sorted by their creation date I guess it's fine :)